src/app/zones/routes.ts

```diff
@@ -16,22 +17,28 @@ export const routes = (
   actions: RouteRecordRaw[],
 ): RouteRecordRaw[] => {
   return [
+    ...actions,
     {
       path: '/zones',
-      name: 'zone-abstract-view',
+      name: 'zone-index-view',
```

Comment:
What's the desired URL structure here?

`/gui/zones/zone-cps` etc etc or `/gui/zone-cps` etc etc ?

(I maybe have some more questions, but not totally sure as I'm still looking but wanted to get this question posted)

Reply:
The desired structure is `/zone-cps`, `/zone-ingresses`, etc. (hence these paths _also_ start with `/`). However, to make sure we’re not accidentally adding an empty path home route I’ve given that one the path `/zones` because it would otherwise be matched *instead* of the home route.

This is a little weird, but I didn’t want to repeat using `ZoneIndexView` for each of the three zone route groups.

Reply:
I am right in thinking you can't actually get to `/zones` right?

I think we should make the URL structure be `/zones/zone-cps` etc, wdyt?

Reply:
Alright, I changed this , too.